plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/messages.properties

```diff
@@ -55,18 +55,18 @@ EsbModelingAssistantProviderTitle=Select domain model element
 EsbModelingAssistantProviderMessage=Available domain model elements:
 Nodes1Group_title=Nodes
 EsbLink1CreationTool_title=ESB Link
-EsbLink1CreationTool_desc=Create New ESB Link
+EsbLink1CreationTool_desc=Manually create a link between two mediators
 NavigatorGroupName_EsbDiagram_1000_links=links
 NavigatorGroupName_ProxyOutputConnector_3002_outgoinglinks=outgoing links
 NavigatorGroupName_ProxyInputConnector_3003_incominglinks=incoming links
 NavigatorGroupName_EsbLink_4001_target=target
 NavigatorGroupName_EsbLink_4001_source=source
 Mediators2Group_title=Mediators
 DropMediator1CreationTool_title=Drop
-DropMediator1CreationTool_desc=Create New Drop Mediator
+DropMediator1CreationTool_desc=Drops a message
```

Comment:
Or we can detail like "Stops the processing of the current message". WDYT?

Reply:
Fixed.